web/source/osk/oskManager.ts

```diff
@@ -167,7 +167,7 @@ namespace com.keyman.osk {
 
       this._Visible = false;  // I3363 (Build 301)
       var s = this._Box.style;
-      s.zIndex='9999'; s.display='none'; s.width='auto';
```

Comment:
I'm not comfortable removing this one if not necessary.

Reply:
On `auto`, the suggestion banner was 50% of the OSK

Reply:
So the `SuggestionBanner` didn't scale with the OSK?  This property _is_ on the OSK itself, not the `Banner`... there should be some setting that could be applied on the Banner's div to make things work.

Reply:
weird. well I reverted, deleted the release/ folder and rebuilt. and the width looks fine